packages/service-worker-mock/models/Response.js

```diff
@@ -2,7 +2,7 @@
 class Response {
   constructor(body, init) {
     this.body = body || '';
-    this.status = (init && init.status) || 200;
+    this.status = (init && typeof init.status === 'number') || 200;
```

Comment:
Looks like we'll either need another `&&` or we can just do a ternary, otherwise we won't actually be getting the value of init.status.
```
this.status = (init && typeof init.status === 'number') ? init.status : 200;
```